<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Script"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionConnect_to_database"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuSyntax_Highlighter">
     <property name="title">
      <string>Syntax Highlighter</string>
     </property>
     <addaction name="actionNone"/>
    </widget>
    <addaction name="actionOpen_Files"/>
    <addaction name="actionFile_Explorer"/>
    <addaction name="actionTable_Of_Contents"/>
    <addaction name="actionSearch_Results"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Whitespace"/>
    <addaction name="actionShow_Line_Endings"/>
    <addaction name="menuSyntax_Highlighter"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionReload_Editor_Preferences"/>
    <addaction name="actionOptions"/>
    <addaction name="actionStart_macro_recording"/>
    <addaction name="actionRun_macro"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuConvert_Line_Breaks">
     <property name="title">
      <string>Convert Line Breaks ...</string>
     </property>
     <addaction name="actionTo_Unix"/>
     <addaction name="actionTo_Windows"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionSelect_Next_Match"/>
    <addaction name="actionSelect_All_Matches"/>
    <addaction name="separator"/>
    <addaction name="menuConvert_Line_Breaks"/>
   </widget>
   <widget class="QMenu" name="menuScript">
    <property name="title">
     <string>Script</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuScript"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Script"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionBuild"/>
   <addaction name="actionRun"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dwExplorer">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>File Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwTOC">
   <property name="windowTitle">
    <string>Table Of Contents</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwOpenFiles">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Open Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="lwOpenFiles"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwSearchResults">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Search Results / Compiler Errors</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew_Script">
   <property name="icon">
    <iconset resource="sideres.qrc">
     <normaloff>:/icons/32/document-new</normaloff>:/icons/32/document-new</iconset>
   </property>
   <property name="text">
    <string>New Script</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/mw/Pictures/tango-icon-theme-0.8.90/32x32/actions/document-open.png</normaloff>C:/Users/mw/Pictures/tango-icon-theme-0.8.90/32x32/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConnect_to_database">
   <property name="text">
    <string>Connect To Database</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="sideres.qrc">
     <normaloff>:/icons/32/system-log-out</normaloff>:/icons/32/system-log-out</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen_Files">
   <property name="text">
    <string>Open Files</string>
   </property>
  </action>
  <action name="actionFile_Explorer">
   <property name="text">
    <string>File Explorer</string>
   </property>
  </action>
  <action name="actionTable_Of_Contents">
   <property name="text">
    <string>Table Of Contents</string>
   </property>
  </action>
  <action name="actionSearch_Results">
   <property name="text">
    <string>Search Results</string>
   </property>
  </action>
  <action name="actionReload_Editor_Preferences">
   <property name="text">
    <string>Reload Editor Preferences</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="sideres.qrc">
     <normaloff>:/icons/32/document-save</normaloff>:/icons/32/document-save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="sideres.qrc">
     <normaloff>:/icons/32/document-save-as</normaloff>:/icons/32/document-save-as</iconset>
   </property>
   <property name="text">
    <string>Save As ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionSelect_Next_Match">
   <property name="text">
    <string>Select Next Match</string>
   </property>
  </action>
  <action name="actionSelect_All_Matches">
   <property name="text">
    <string>Select All Matches</string>
   </property>
  </action>
  <action name="actionTo_Unix">
   <property name="text">
    <string>To Unix</string>
   </property>
  </action>
  <action name="actionTo_Windows">
   <property name="text">
    <string>To Windows</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionShow_Whitespace">
   <property name="text">
    <string>Show Whitespace</string>
   </property>
  </action>
  <action name="actionShow_Line_Endings">
   <property name="text">
    <string>Show Line Endings</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="icon">
    <iconset resource="sideres.qrc">
     <normaloff>:/icons/32/edit-clear</normaloff>:/icons/32/edit-clear</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="sideres.qrc">
     <normaloff>:/icons/32/media-playback-start</normaloff>:/icons/32/media-playback-start</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionStart_macro_recording">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Record macro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Y</string>
   </property>
  </action>
  <action name="actionRun_macro">
   <property name="text">
    <string>Run macro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sideres.qrc"/>
 </resources>
 <connections/>
</ui>
